mantle/network/ntp/protocol_test.go

```diff
@@ -97,7 +97,7 @@ func TestHeaderUnmarshal(t *testing.T) {
 			t.Errorf("testData[%d].Raw is invalid", i)
 		}
 		h := Header{}
-		h.UnmarshalBinary(d.Raw)
+		_ = h.UnmarshalBinary(d.Raw)
```

Comment:
I think we should check this error.

Reply:
Thank you for review. Updated and checked the errors.